README.md

```diff
@@ -24,9 +24,19 @@ component freshness. Workflow Manager will regularly check your cluster against
 the latest stable components. If components are missing due to failure or are
 simply out of date, Workflow operators will know at a glance.
 
+By default, Workflow Manager will make version checks by default. This submits
```

Comment:
Two usages of "default" in this sentence. I have a preference for "By default, Workflow Manager will make version checks." It seems a bit more explicit, which is what we want in this case.